docs/application/native/guides/app-management/data-control.md

```diff
@@ -46,11 +46,13 @@ To enable your application to use the data control functionality:
 
 2. For the provider, in the Tizen Studio, double-click **tizen-manifest.xml**, and in the manifest editor, go to **Advanced > Data Control**, and click **+** to add the provider details. Add the **Read** and **Write** access rights to both **SQL** and **Map** types.
 
-   The following code example shows how the `<datacontrol>` element is consequently added to the `tizen-manifest.xml` file:
+    You can set the data access to be trusted, allowing other applications signed with the same certificate to access the data. You can also define privileges to restrict access to applications having the specific privileges.
```

Comment:
You can set the data access to be trusted, allowing other applications signed with the same certificate to access the data. You can also define privileges to restrict access to applications having the specific privileges.

[Surbhi] Modified the content to:

You can set the data access to trusted, to allow other applications that are signed with the same certificate to access the data.
Additionally, you can also define privileges to restrict access for applications having certain predefined privileges.